modin/core/execution/dask/implementations/pandas_on_dask/partitioning/partition.py

```diff
@@ -308,7 +300,7 @@ def ip(self):
         return self._ip_cache
 
 
-def apply_func(partition, func, *args, **kwargs):
+def apply_func(partition, func, f_args, f_kwargs):
```

Comment:
```suggestion
def apply_func(partition, func, *args, **kwargs):
```
Why doesn't this work?

Reply:
Doing this made it easier to pass arguments to `DaskWrapper.deploy`, as [here](https://github.com/modin-project/modin/pull/4715/files/0939080168bad7f612e704adc45178401b9b2093#diff-9982d8740602ab4ffe8104fb6590b98d10b065e8731fa2cc4b286bc1e84c20d5R158), and this change fixed the performance difference between this PR and master on my micro-benchmark (calling `df.describe()` on a 2^16 x 2^12 dataframe). I'll refactor it to your version for consistency with Ray.